tests/gem5/gpu/test_gpu_pannotia.py

```diff
@@ -0,0 +1,344 @@
+    config_args=[
+        "-n3",
+        "--mem-size=8GB",
+        "-c",
+        joinpath(binary_path, "bc.gem5"),
+        "--options",
+        joinpath(dataset_path, "1k_128k.gr"),
+    ],
+    valid_isas=(constants.vega_x86_tag,),
+    valid_hosts=(constants.host_gcn_gpu_tag,),
+    length=constants.very_long_tag,
+)
+
+gem5_verify_config(
+    name="gpu-apu-se-pannotia-bc-2k-1M",
+    fixtures=(),
+    verifiers=(),
+    config=joinpath(config.base_dir, "configs", "example", "apu_se.py"),
+    config_args=[
+        "-n3",
+        "--mem-size=8GB",
+        "-c",
+        joinpath(binary_path, "bc.gem5"),
+        "--options",
+        joinpath(dataset_path, "2k_1M.gr"),
+    ],
+    valid_isas=(constants.vega_x86_tag,),
+    valid_hosts=(constants.host_gcn_gpu_tag,),
+    length=constants.very_long_tag,
+)
+
+gem5_verify_config(
+    name="gpu-apu-se-pannotia-color-maxmin-ecology",
+    fixtures=(),
+    verifiers=(),
+    config=joinpath(config.base_dir, "configs", "example", "apu_se.py"),
+    config_args=[
+        "-n3",
+        "--mem-size=8GB",
+        "-c",
+        joinpath(binary_path, "color_maxmin.gem5"),
+        "--options",
+        f'{joinpath(dataset_path, "ecology1.graph")} 0',
+    ],
+    valid_isas=(constants.vega_x86_tag,),
+    valid_hosts=(constants.host_gcn_gpu_tag,),
+    length=constants.very_long_tag,
+)
+
+gem5_verify_config(
+    name="gpu-apu-se-pannotia-color-maxmin-g3-circuit",
+    fixtures=(),
+    verifiers=(),
+    config=joinpath(config.base_dir, "configs", "example", "apu_se.py"),
+    config_args=[
+        "-n3",
+        "--mem-size=8GB",
+        "-c",
+        joinpath(binary_path, "color_maxmin.gem5"),
+        "--options",
+        f'{joinpath(dataset_path, "G3_circuit.graph")} 0',
+    ],
+    valid_isas=(constants.vega_x86_tag,),
+    valid_hosts=(constants.host_gcn_gpu_tag,),
+    length=constants.very_long_tag,
+)
+
+gem5_verify_config(
+    name="gpu-apu-se-pannotia-color-max-ecology",
+    fixtures=(),
+    verifiers=(),
+    config=joinpath(config.base_dir, "configs", "example", "apu_se.py"),
+    config_args=[
+        "-n3",
+        "--mem-size=8GB",
+        "-c",
+        joinpath(binary_path, "color_max.gem5"),
+        "--options",
+        f'{joinpath(dataset_path, "ecology1.graph")} 0',
+    ],
+    valid_isas=(constants.vega_x86_tag,),
+    valid_hosts=(constants.host_gcn_gpu_tag,),
+    length=constants.very_long_tag,
+)
+
+gem5_verify_config(
+    name="gpu-apu-se-pannotia-color-max-g3-circuit",
+    fixtures=(),
+    verifiers=(),
+    config=joinpath(config.base_dir, "configs", "example", "apu_se.py"),
+    config_args=[
+        "-n3",
+        "--mem-size=8GB",
+        "-c",
+        joinpath(binary_path, "color_max.gem5"),
+        "--options",
+        f'{joinpath(dataset_path, "G3_circuit.graph")} 0',
+    ],
+    valid_isas=(constants.vega_x86_tag,),
+    valid_hosts=(constants.host_gcn_gpu_tag,),
+    length=constants.very_long_tag,
+)
+
+gem5_verify_config(
```

Comment:
same for FW: https://github.com/gem5/gem5/blob/stable/tests/deprecated/weekly.sh#L309